nixos/modules/system/boot/systemd.nix

```diff
@@ -597,6 +597,11 @@ in
       # drop-in in /etc, it does apply.
       overrideStrategy = "asDropin";
     };
+    systemd.services."systemd-mkswap@" = {
```

Comment:
Shouldn't this be conditional on swapDevices?

Reply:
Ensuring systend-mkswap@ works in all cases is ok to have unconditionally. util-linux is probably in the runtime closure of the system anyways.